File: week-03/day-05/01.py
# Create a `Circle` class that takes it's radius as cinstructor parameter
# It should have a `get_circumference` method that returns it's circumference
# It should have a `get_area` method that returns it's area

class Circle:

    def circumference(radius):
        a = radius * 2 *3.14
        return a

    def get_area(radius):
        a = radius ** 2 *3.14
        return a


print(Circle.get_area(30))
print(Circle.circumference(30))

File: week-08/day-01/01.js
'use strict';

// Create a constructor for creating Animals.
// it should take one parameter: what the animal says
// Every animal should have a method called say() that prints what the animal says


// var Animal = new Animals();

function Animals(says) {
   this.says = says;
}

Animals.prototype.say = function(){
   console.log(this.says);
}

var dog = new Animals("vhao");
var cat = new Animals("meow");

dog.say();
cat.say();
